src/configuration-store/configuration-store.ts

```diff
@@ -26,7 +26,11 @@ export interface IConfigurationStore<T> {
   getKeys(): string[];
   isInitialized(): boolean;
   isExpired(): Promise<boolean>;
-  setEntries(entries: Record<string, T>): Promise<void>;
+  setEntries(entries: Record<string, T>): Promise<boolean>;
```

Comment:
setEntries now returns `Promise<boolean>`. If you resolve with true, it tells `js-client-sdk-common` to update the `configFetchTime` and `configPublishTime`